dapps/marketplace/src/index.js

```diff
@@ -1,8 +1,19 @@
 // Ensure storage is cleared on each deploy
 const appHash = process.env.GIT_COMMIT_HASH || 'marketplace'
 const ognNetwork = localStorage.ognNetwork
+
 if (localStorage.appHash !== appHash) {
+  let exceptions = ['growth_auth_token']
+  exceptions = exceptions
+    .map(key => {
```

Comment:
can make it 1 line with `.map(key => ({ key, value: localStorage.getItem(key) })` 😊

Reply:
nice thanks! fixed